dkandalintsev/login.feature

```diff
@@ -0,0 +1,127 @@
+    And users's Google accounts have this credentials:
+        | email address               | phone number | password |
+        | testkandalintsev@gmail.com  | +79819139630 | Test123! |
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    When user clicks "Sign in with Google" button
+    Then user is on the Quantori Scoring main page
+
+Scenario: 1.03 User has multiple saved Google accounts on device and tries to sign in
+    Given user is signed in with two Google accounts on google.com in the web browser
+    And users's Google accounts have this credentials:
+        | email address               | phone number | password |
+        | testkandalintsev@gmail.com  | +79819139630 | Test123! |
+        | testkandalintsevd@gmail.com | +79646149307 | Test123! |
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    When user clicks "Sign in with Google" button
+    Then list of user's existing Google accounts is shown
+
+Scenario: 1.04 User signs in with multiple saved and logged in Google accounts
+    Given user has logged in Google accounts in browser
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    And user clicked "Sign in with Google" button
+    When user clicks on one of his Google accounts in the list
+    Then user is on the Quantori Scoring main page
+
+Scenario: 1.05 User has saved Google account, but signed out
+    Given user is logged out of his Google account
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    And user clicked "Sign in with Google" button
+    When user clicks on one of his Google accounts in the list
+    Then user needs to fill in password field for his Google account
+
+Scenario: 1.06 User signs in with saved and logged out Google account
+    Given user is logged out of his Google account
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    And user clicked "Sign in with Google" button
+    And user filled in the password field with valid <password>
+    When user clicks "Next"
+    Then user is on the Quantori Scoring main page
+
+Scenario: 1.07 User does not use right password for saved and logged out Google account
+    Given user is logged out of his Google account
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    And user clicked "Sign in with Google" button
+    And user filled in the password field with invalid <password>
+    When user clicks "Next"
+    Then user sees that password input is red
+    And user has "Wrong password. Try again or click ‘Forgot password’ to reset it." message uder the password input
+
+Scenario: 1.08 User checks if signing in without saved Google accounts is possible
+    Given user does not have saved Google accounts in browser
+    And user is on the https://qa.ddso-spot.quantori.com/ page
+    When user clicks "Sign in with Google" button
+    Then user sees "Email or phone" input for Google account
+
+Background:
```

Comment:
Yes, if you need to use "Background" it should be placed at the beginning of the feature file and your background will be used for all scenarios in your feature file. 
Or, if you understand that the next scenarios should have the Background, you can separate one feature file from the two feature files.